### ISSUE TITLE
Typescript typings missing `ignoreJSONStructure`

### ISSUE THREAD

**Issue by danielweck** [i18next/i18next#1589]
Source:

https://github.com/i18next/i18next/blob/f5ca8f6f4b228bc6ce8b9b8640eec30e99044397/index.d.ts#L216

...in relation to this code change:

https://github.com/i18next/i18next/pull/1584

See changelog:

https://github.com/i18next/i18next/blob/master/CHANGELOG.md#2000

**Comment by adrai** [comment]
Do you like to create a PR?

**Comment by danielweck** [comment]
Hello, I currently do not have the bandwidth, sorry.

My workaround is:
```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
ignoreJSONStructure: false
```

**Comment by adrai** [comment]
//cc @pedrodurek

**Comment by adrai** [comment]
will be included in the next release

**Comment by danielweck** [comment]
Wow, `stale` bot kicks-in after 7 days only ! :) 

![26350515](https://user-images.githubusercontent.com/625201/113450314-84d9bf00-93f7-11eb-98bb-4970b466fecd.png)

**Comment by adrai** [comment]
included in >= v20.2.0
